src/grpc/stream_infer_handler.cc

```diff
@@ -370,9 +379,13 @@ ModelStreamInferHandler::Process(InferHandler::State* state, bool rpc_ok)
 
     next_read_state->context_->responder_->Read(
         &next_read_state->request_, next_read_state);
-
+  } else if (state->step_ == Steps::PARTIAL_COMPLETION) {
+    state->step_ = Steps::COMPLETE;
```

Comment:
Similar to [other thread](https://github.com/triton-inference-server/server/pull/6345/files#r1337660823), how does this state get put back on queue to process future transitions, assuming cq_->Next() is popping them off the queue?

Reply:
AsyncNotifyWhenDone()/Finish() call will put the state back to the compelition queue.